pkg/event/mock_test.go

```diff
@@ -3,6 +3,6 @@ package event
 //go:generate mockgen -destination=./mock/event.go -package=mock -mock_names=MuxComponent=MuxComponent,Component=Component,AdminComponent=AdminComponent,ConsoleModule=ConsoleModule,StatisticModule=StatisticModule github.com/cloudtrust/keycloak-bridge/pkg/event MuxComponent,Component,AdminComponent,ConsoleModule,StatisticModule
 //go:generate mockgen -destination=./mock/dbmodule.go -package=mock -mock_names=EventsDBModule=EventsDBModule github.com/cloudtrust/common-service/database EventsDBModule
 //go:generate mockgen -destination=./mock/instrumenting.go -package=mock -mock_names=Histogram=Histogram,Metrics=Metrics github.com/cloudtrust/common-service/metrics Histogram,Metrics
-//go:generate mockgen -destination=./mock/logging.go -package=mock -mock_names=Logger=Logger github.com/go-kit/kit/log Logger
+//go:generate mockgen -destination=./mock/logging.go -package=mock -mock_names=Logger=Logger github.com/cloudtrust/common-service/log Logger
```

Comment:
Do we really want to mock loggers ? (We could use NoopLogger rather than a mocked logger)

Reply:
Agree